test/bbcNewsSearch.js

```diff
@@ -0,0 +1,44 @@
+const {Builder,By,Key, until} = require("selenium-webdriver");
```

Comment:
What do you need this file for?
It is a complete copy of the one located in the `selenium_testing` folder.

Reply:
test folder was removed. seems like side effect of copying or smth like that. It's not necessary